src/backend/wayland/application.rs

```diff
@@ -112,6 +112,12 @@ impl Application {
             |it| Ok(Some(it)),
         )?;
 
+        let xkb_context = Context::new();
+        xkb_context.set_log_level(
+            tracing::level_filters::STATIC_MAX_LEVEL
+                .into_level()
+                .unwrap_or(tracing::Level::TRACE),
+        );
```

Comment:
You end up deleting this log level in a subsequent commit, but leave `xkb_context` as a separate var, which led me to wonder where it might be used outside of the constructor, but then it wasn't.

Could we just kill it from here instead?